source/documentation/testing-process.html.md.erb

```diff
@@ -1,13 +1,33 @@
 ---
 title: Testing process
-weight: 15
+weight: 30
 ---
 
 # Testing process
 
-This is a prototype testing guide used for research. The testing process it describes is still in development.
-
 You must test the compatibility of your software with [EMCS API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/excise-movement-control-system-api/1.0).
 
-You will be able to connect to the [API in External Test]().
+After [obtaining a bearer token](obtain-a-bearer-token.html), use postman or curl to call the API at: <br></br> `https://test-api.service.hmrc.gov.uk/customs/excise/movements`
+
+The following values in angled brackets (<>) must be replaced:
```

Comment:
To quote a wise man named J. Liddell, 
"it seems like the convention in HMRC is to use square brackets and caps (e.g. [YOUR-CLIENT-ID]) for things the user needs to replace"

Reply:
Corrected